### PR TITLE
fix: set default TTL for APCu cache as per docs

### PR DESCRIPTION
**Pull request by icewind1991**
Make the method behavior match the description

### CODE REVIEW

**Review by sorbaugh — APPROVED** [review]

File: lib/public/ICache.php [hunk 1]
```diff
@@ -15,6 +15,8 @@
  * @since 6.0.0
  */
 interface ICache {
+	const DEFAULT_TTL = 24 * 60 * 60;
```
Comment (nickvergessen) [inline lib/public/ICache.php:18]:
```suggestion
	/**
	 * @since 30.0.0
	 */
	const DEFAULT_TTL = 24 * 60 * 60;
```

**Review by nickvergessen — APPROVED** [review]
Memcached seems to have the same problem?

**Review by joshtrichards — COMMENTED** [review]
Couple thoughts:

- Should we do this for `inc` and `dec` too? e.g. https://www.php.net/manual/en/function.apcu-inc.php
- Should we adjust the logging wrapper to include the ttl at the same time?
- Probably out of scope, but may also be an opportunity clean-up the no longer needed code for an early PHP 7 bug in `inc` & `dec` a bit farther down

**Comment by icewind1991** [comment]
- clarified the documentation for `inc` and `dec` and cleanup up the implementation now that all supported versions behave the same

**Review by ChristophWurst — APPROVED** [review]

**Review by Altahrim — APPROVED** [review]

**Comment by SystemKeeper** [comment]
> * clarified the documentation for `inc` and `dec` and cleanup up the implementation now

Is it correct that you removed the comment from `dec` but did not change the implementation? (Only for `inc`)?

**Review by juliusknorr — APPROVED** [review]

**Comment by solracsf** [comment]
This is causing troubles on v30: https://github.com/nextcloud/server/issues/48007

**Pull request merged**